client/containers/Pub/SpubHeader/SpubHeaderToolBar/SpubHeaderToolbar.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { Dispatch, SetStateAction, useMemo } from 'react';
+import { Tab, Tabs, TabId, Icon, IconName, Button } from '@blueprintjs/core';
+import Color from 'color';
+
+import { GridWrapper } from 'components';
+import { SubmissionStatus } from 'types';
+import { usePageContext, usePendingChanges } from 'utils/hooks';
+
+require('./spubHeaderToolbar.scss');
+
+const renderTabTitle = (icon: IconName, title: string) => (
+	<>
+		<Icon icon={icon} /> {title}
+	</>
+);
+
+type Props = {
+	selectedTab: TabId;
+	onSelectTab: Dispatch<SetStateAction<TabId>>;
+	status: SubmissionStatus;
+	showSubmitButton: boolean;
+	onSubmit: () => any;
+};
+
+const SpubHeaderToolbar = (props: Props) => {
+	const instructionTabTitle = renderTabTitle('align-left', 'Instructions');
+	const submissionTabTitle = renderTabTitle('manually-entered-data', 'Submission');
+	const contributorsTabTitle = renderTabTitle('people', 'Contributors');
+	const previewTabTitle = renderTabTitle('eye-open', 'Preview');
+	const maybeActiveClass = (tabId: string) =>
+		`${tabId === props.selectedTab ? 'active' : 'inactive'}`;
+
+	const { communityData } = usePageContext();
+
+	const lighterAccentColor = useMemo(
+		() => Color(communityData.accentColorLight).alpha(0.2),
+		[communityData.accentColorLight],
+	);
+
+	const darkerAccentColor = useMemo(
+		() => Color(communityData.accentColorDark),
+		[communityData.accentColorDark],
+	);
+	const { pendingCount } = usePendingChanges();
+	const isSaving = pendingCount > 0;
+
+	const status = isSaving ? (
+		<strong>
+			<em>Saving</em>
+		</strong>
+	) : (
+		<p className="status-text">{props.status}</p>
+	);
+
+	const renderRight = props.showSubmitButton ? (
+		<Button className="submission-button" outlined={true} onClick={props.onSubmit}>
+			Submit
+		</Button>
+	) : (
+		<div className="status">
+			<em>status:&nbsp;&nbsp;&nbsp;</em>
+			<strong>{status}</strong>
+		</div>
+	);
+	return (
+		<div style={{ background: lighterAccentColor, color: darkerAccentColor }}>
+			<GridWrapper containerClassName="gridParent">
+				<div className="spubheader-toolbar">
+					<Tabs
+						id="spubHeader"
+						onChange={props.onSelectTab}
+						selectedTabId={props.selectedTab}
+						className="spub-header-component tabs bp3-large"
```

Comment:
I'm skeptical that you need any of these classes here:

- `.spub-header-component` is now the parent component and is dragging in some unwanted background color in Storybook. Guessing this is coming from the tabs in `SpubHeader.tsx`. <kbd>Cmd + C</kbd> <kbd>Cmd + V</kbd>...great power, great responsibility, etc.
- `.tabs` is being used to select the component in the SCSS, but you could just as easily use the `.bp3-tabs`
- `.bp3-large` is doing the right thing, but you should use the `large` prop on `Tabs` instead and let it render the appropriate class to make it big.